nyxx/lib/src/core/channel/ThreadPreviewChannel.dart

```diff
@@ -0,0 +1,58 @@
+part of nyxx;
+
+/// Given when a thread is created as only partial information is available. If you want the final channel use [getThreadChannel]
+class ThreadPreviewChannel extends IChannel {

```

Comment:
Maybe implementing TextChannel on ThreadPreview would be nice. I mean for most functionallity we only need id which is already there

Reply:
done